metricbeat/module/mongodb/oplog/_meta/fields.yml

```diff
@@ -0,0 +1,33 @@
+- name: oplog
+  type: group
+  description: >
+    oplog provides an overview of replication status, which is retrieved from db.getReplicationInfo().
+  fields:
+    - name: size
+      type: gorup
```

Comment:
`group`

Reply:
Hurray!
I think there should be a tool to verify `fields.yml` structure. Something like [YAML lint](yamllint.com).

Reply:
Yes, we should probably try to do some validations so these errors can be detected before.